tests/base_test_mixins.py

```diff
@@ -513,3 +690,41 @@ def debug_print_response_messages(self, response):
             print("-----------message end-------------\n")
         except KeyError:
             print("\n-------no message----------")
+
+
+class StaticDockerContainerTestMixin(object):
```

Comment:
I find this name misleading, since there's no Docker involved at all. `InProcessFakeContainer`?

Reply:
(I.e. I don't necessary like "static" either, as an adjective.)